app/commands/remove_place.rb

```diff
@@ -0,0 +1,35 @@
+class RemovePlaceCommand < BaseCommand
+  def initialize(user, args)
+    @user = user
+
+    matches = args.match(RemovePlaceCommand.regex)
+    @name = matches[:name]
```

Comment:
ah, it is, isn't it.

what do you think? i kinda like the idea of passing it known options (ie passing the matches in), but also like the idea of not changing too much.